### PR TITLE
Guidelines FAIR Principles

### PR DESCRIPTION
**Pull request by nilshempelmann**
might need to add technical details, since it is a birdhouse guide. e.g thredds  server 
Was using text that we wrote for the publication, so the work is not lost.

### CODE REVIEW

File: docs/source/guide_fair.rst [hunk 1]
```diff
@@ -1,22 +1,81 @@
...
 
 .. contents::
     :local:
     :depth: 1
 
-.. _fairdata:
 
-FAIR Data
----------
+Climate datasets rapidly grow in volume and complexity and creating climate products requires high bandwidth, massive storage and large compute resources. For some regions, low bandwidth constitutes a real obstacle to developing climate services. Data volume also hinders reproducibility because very few institutions have the means to archive original data sets over the long term. Moreover, typical climate products often aggregate multiple sources of information, yet mechanisms to systematically track and document the provenance of all these data are only emerging. So although there is a general consensus that climate information should follow the **FAIR Principles** :cite:`Wilkinson2016,mons2017`, that is be *findable, accessible, interoperable, and reusable*, a number of obstacles hinder progress. Following guidelines might help to set up climate services information systems to avoid inefficiency and enabling synergies. The four FAIR Principles aspects can be seen in line with **data** as well as the processing **structures**.
```
Comment (huard) [inline docs/source/guide_fair.rst:11]: The following guidelines can help set up efficient climate services information systems, and show how the four FAIR Principles not only apply to data, but also to analytical processes.

File: docs/source/guide_fair.rst [hunk 2]
```diff
@@ -1,22 +1,81 @@
...
-.. note:: Work in progress
+.. _findable:
 
-.. _fairstructure:
+Findable
+--------
 
-FAIR Structure
---------------
+Findable is the basic requirement for data and product usage and already an difficult obstacle with time intensive work for the data provider ensuring find-able data. On the production level finding algorithms requires open source software with intensive documentation.
```
Comment (huard) [inline docs/source/guide_fair.rst:18]: I would reuse an existing definition from a reference, and tweak it to include algorithms.

File: docs/source/guide_fair.rst [hunk 3]
```diff
@@ -1,22 +1,81 @@
...
 
-.. note:: Work in progress
+**Finding data:**
```
Comment (huard) [inline docs/source/guide_fair.rst:20]: How does this relate to birdhouse? I'm not sure I see the value of discussing the FAIR principles if we do not show how they are embodied in this project?

Comment (nilshempelmann) [reply docs/source/guide_fair.rst:20]: It is a guideline for orientation, when new structures are being designed. If possible we should include technical birdhouse solutions to be closer to the project. The text to too general for the moment.

**Comment by nilshempelmann** [comment]
@huard @cehbrecht I restructured the guidelines. May be parts can be moved to the new tutorial section. 
Gues for now it is fine but the guidelines become interesting when other big projects are upcoming.

File: docs/source/guide_fair.rst (hunk 3 above)
Comment (nilshempelmann) [reply docs/source/guide_fair.rst:20]: Data repository design including finding dissemination of data is part of the eventually upcoming project

File: docs/source/guide_fair.rst (hunk 2 above)
Comment (nilshempelmann) [reply docs/source/guide_fair.rst:18]: https://www.nature.com/articles/sdata201618 refers also to algorithms

File: docs/source/tutorials.rst [hunk 4]
```diff
@@ -6,3 +6,160 @@ Tutorials
...
+.. todo:: Python tutorial
+
+Climate data management
+-----------------------
+**general Tutorial** about climate and related data handling.
+
+* esfg-python client
+
+
+WPS general usage
```
Comment (cehbrecht) [inline docs/source/tutorials.rst:22]: There is also an outdated pywps/birdhouse workshop which could be linked:
https://birdhouse-workshop.readthedocs.io/en/latest/

**Review by cehbrecht — COMMENTED** [review]
Currently the technical details are more up-to-date in the cookiecutter template. We might not need to describe them in the birdhouse docs ... they get so easily outdated. So, keeping the birdhouse-docs for a common overview and organizational things ... links to meetings etc ...

File: docs/source/tutorials.rst (hunk 4 above)
Comment (nilshempelmann) [reply docs/source/tutorials.rst:22]: Great! 
That's going to be a section in the tutorials

**Pull request merged**